### PR TITLE
Don't allow block deletion in the flyout.  Make copy-paste always copy to the main workspace

### PR DESCRIPTION
**Pull request by rachel-fenichel**
## The basics

- [x] I branched from develop
- [x] My pull request is against develop
- [x] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)

### Resolves

Fixes https://github.com/LLK/scratch-blocks/issues/1373

### Proposed Changes

On keyboard paste, paste to the target workspace.
On keyboard cut, backspace, and delete, block the action if the selected block was in the flyout.

### Reason for Changes

See https://github.com/LLK/scratch-blocks/issues/1373

You could delete and paste blocks in the flyout, which was lovely.

### Test Coverage

Tested on:
* Desktop Chrome

### Additional Information

@kchadha fyi

### CODE REVIEW

**Comment by rachel-fenichel** [comment]
This bug is somehow new behaviour--it doesn't happen on blockly games.  But not too new, because it's on scratch-blocks.

**Comment by amazinigmech2418** [comment]
I think the ctrl. + v is the problem. It should make a copy and place it at the mouse pointer if the mouse pointer isn't in the menu.

**Comment by rachel-fenichel** [comment]
Blockly doesn't attach blocks to the pointer on paste.  Also, we don't know the pointer location during a key event (without additional work to track it continuously).

**Comment by amazinigmech2418** [comment]
@rachel-fenichel You could use a mousemove event to read the mouse position when pasting

**Comment by rachel-fenichel** [comment]
This issue came up because my dragging work made it possible to select a block in the flyout.  Previously `Blockly.selected` would not be set by interacting with a block in the flyout.

File: core/workspace_svg.js [hunk 1]
```diff
@@ -267,6 +267,13 @@ Blockly.WorkspaceSvg.prototype.flyoutButtonCallbacks_ = {};
...
 Blockly.WorkspaceSvg.prototype.toolboxCategoryCallbacks_ = {};
 
+/**
+ * In a flyout, the target workspace where blocks should be placed after a drag.
+ * Otherwise null.
+ * @type {?Blockly.WorkspaceSvg}
+ * @package
+ */
+Blockly.WorkspaceSvg.prototype.targetWorkspace = null;
 /**
```
Comment (AnmAtAnm) [inline core/workspace_svg.js]: Nit: empty line before documentation

Comment (rachel-fenichel) [reply core/workspace_svg.js]: Fixed

**Pull request merged**